.ci/gcb-generate-diffs.yml

```diff
@@ -191,9 +216,11 @@ steps:
           - $BUILD_ID
           - $PROJECT_ID
           - "19"  # Build step
```

Comment:
could you update the build step numbers? Since a couple steps are being added higher up these will likely be incorrect.

Reply:
... it might make more sense to just accept these will be briefly broken & update them in a future PR after everything is cleaned up.

Reply:
These numbers are somehow correct? https://pantheon.corp.google.com/cloud-build/builds/8d5da692-d4c5-4b57-b63a-fa44eb84ea3d?project=graphite-docker-images&e=13803378&jsmode=O&mods=monitoring_api_prod
Since there is a complicated dependency graph, not sure what to number them.